sdk/deviceupdate/azure-iot-deviceupdate/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## 1.0.0-beta.2 (Unreleased)
```

Comment:
Please use this changelog format - https://azure.github.io/azure-sdk/policies_releases.html#change-logs